x/wasm/ibc_testing/chain.go

```diff
@@ -334,10 +334,11 @@ func (chain *TestChain) AddTestConnection(clientID, counterpartyClientID string)
 // format:
 // connectionid<index>
 func (chain *TestChain) ConstructNextTestConnection(clientID, counterpartyClientID string) *TestConnection {
-	connectionID := ConnectionIDPrefix + strconv.Itoa(len(chain.Connections))
+	connectionID := chain.ChainID + ConnectionIDPrefix + strconv.Itoa(len(chain.Connections))
```

Comment:
ChainID is here just to make them unique for debugging tests, right? And not needed by the protocol?

If they have to be globally unique over all chains, good to document that

Reply:
(Good to ensure difference in test, so we don't accidentally mix up local and remote ids)

Reply:
yes, the intend was to make them unique for debugging when working with 2 chains. Protocol wise they don't have to be globally unique